### PR TITLE
BENCH-134: Adjusted Product and Category DTOs

### PR DESCRIPTION
**Pull request by harrystead2**

### CODE REVIEW

File: src/dtos/CategoryDto.ts [hunk 1]
```diff
@@ -1,4 +1,5 @@
 export interface CategoryDto {
   id: number;
   name: string;
```
Comment (mbaltrimas) [inline src/dtos/CategoryDto.ts]:
```
    Answer.King.Domain.Orders.Models.Category:
      type: object
      properties:
        id:
          type: string
          format: uuid
        name:
          type: string
          nullable: true
        description:
          type: string
          nullable: true
      additionalProperties: false
```

according to the spec the name should be nullable

File: src/dtos/ProductCreateDto.ts [hunk 2]
```diff
@@ -0,0 +1,5 @@
+export interface ProductCreateDto {
+  name?: string;
+  price: number;
+  description?: string;
+}
```
Comment (mbaltrimas) [inline src/dtos/ProductCreateDto.ts]:
```
    Answer.King.Api.RequestModels.ProductDto:
      type: object
      properties:
        name:
          type: string
          nullable: true
        description:
          type: string
          nullable: true
        price:
          type: number
          format: double
        category:
          $ref: '#/components/schemas/Answer.King.Api.RequestModels.CategoryId'
      additionalProperties: false
```

seems like the api spec also wants the category for creating products

**Review by mbaltrimas — CHANGES_REQUESTED** [review]

File: src/dtos/ProductCreateDto.ts (hunk 2 above)
Comment (harrystead2) [reply src/dtos/ProductCreateDto.ts]: Don't think the create product model is really necessary tbh

**Comment by mbaltrimas** [comment]
looks good! you might want to merge into `develop` btw

**Pull request merged**